src/vs/workbench/contrib/search/browser/search.contribution.ts

```diff
@@ -382,6 +382,11 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('search.experimental.quickAccess.preserveInput', "Controls whether the last typed input to Quick Search should be restored when opening it the next time."),
 			'default': false
 		},
+		'search.experimental.quickAccess.seedFileSearchStringFromSelection': {
```

Comment:
I don't think this needs to be an experimental setting. I think `search.quickOpen.seedStringFromSelection` should be sufficient.

Reply:
Got it, will update that